Sources/TuistDependencies/DependenciesController.swift

```diff
@@ -121,24 +126,26 @@ public final class DependenciesController: DependenciesControlling {
 
         var dependenciesGraph = DependenciesGraph(thirdPartyDependencies: [:])
 
-        if let carthageDepedencies = dependencies.carthage, !carthageDepedencies.dependencies.isEmpty {
-            dependenciesGraph = try carthageInteractor.install(
+        if let carthageDependencies = dependencies.carthage, !carthageDependencies.dependencies.isEmpty {
```

Comment:
Thanks for fixing typos and sorry that I left them. 🙄

Reply:
no prob 🙌  I have fixed also some other similar ones 👍